perfkitbenchmarker/analyze.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
```

Comment:
Think about this again, the name "analyze.py" is just way too generic. This class was made for object storage benchmarks, and lots of its logic was designed for the object storage multi-stream throughput analysis, at least for now.

Please either move this to the same directory where object storage benchmark lives, or rename it to something like object_storage_multistream_analyze.py.